frontend/src/components/common/ItemSortMethod.tsx

```diff
@@ -18,6 +19,22 @@ type ItemSortMethodProps = {
 
 const ItemSortMethod = ({ onChange }: ItemSortMethodProps) => {
   const { formatMessage } = useIntl();
+  const themeContext = useContext(ThemeContext);
+
+  const customStyles = {
+    menu: (styles: any) => ({
+      ...styles,
+      background: `${themeContext.theme === LIGHT ? "#ffffff" : "var(--dark)"}`,
```

Comment:
Instead of checking the theme like this, it is possible to use the css variable --light-color, it takes the color ```#f2f2f2``` when the light mode is active, and ```#333333``` when the dark mode is active.

it will became ```background: "var(--light-color)"```